sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClientLiveTests.cs

```diff
@@ -86,6 +86,10 @@ public async Task CreateMetricAnomalyDetectionConfiguration()
 
             Assert.That(getConfig.Id, Is.EqualTo(createdConfiguration.Id));
 
+            getConfig.Description = "updated";
+
+            await adminClient.UpdateMetricAnomalyDetectionConfigurationAsync(getConfig.Id.ToString(), getConfig);
```

Comment:
Why the need for `ToString()`? I might be looking in the wrong place but => https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AnomalyDetection/MetricAnomalyDetectionConfiguration.cs#L40

Reply:
Good catch - this used to be a `Guid`